src/pagination/Pagination.jsx

```diff
@@ -74,7 +74,7 @@ const StyledPagesTruncation = styled('span')`
 function Pagination({ totalPages, activePage, onPageClick, getPageUrl }) {
   const visiblePieces = computeVisiblePieces(totalPages, activePage)
 
-  if (totalPages === 1) {
+  if (totalPages === 0 || totalPages === 1) {
```

Comment:
Optional: could be slightly simplified
```suggestion
  if (totalPages < 2) {
```